backend/src/models/product.py

```diff
@@ -1,13 +1,17 @@
 from pydantic import BaseModel, Field
 from bson import ObjectId
-from typing import List
+from typing import List,Optional
 
 class ProductModel(BaseModel):
     id: str = Field(default_factory=lambda: str(ObjectId()), alias="_id")
     product_name: str
     description: str
     price: float
+    seller_id:  Optional[str] = None
```

Comment:
Does this show seller specific products?

Reply:
Yes Durga, this field is used to handle the task where an seller can see only the products that are uploaded by them, not other seller products

Reply:
hoo kk, looks good merge it now!!!